reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -72,29 +90,44 @@
   padding: 7px;
   width: 7em;
   height: 7em;
+  color: #000000;
+}
+
+.sonification-play-button {
+  background-color: inherit;
+  border-color: #cccccc;
+}
+
+.sonification-play-button h5 {
+  font-size: 0.75rem;
+  color: #6c6c6c;
+  margin-bottom: 0;
+}
+
+.darkmode .sonification-play-button h5 {
+  color: #f2f2f2;
 }
 
 .sonification-play-button:hover {
   box-shadow: 5px 5px 5px #aaaaaa;
 }
 
+.darkmode .sonification-play-button:hover,
+.darkmode .sonification-play-button h5:hover {
+  box-shadow: none;
```

Comment:
Why no box-shadow on dark mode but we have it on light mode?

Reply:
@rcarmitage All shadows have been removed in dark mode in favour of a more flat design. If we don't specify "box-shadow: none," the shadow will be inherited from the light mode.

Reply:
Matter of etiquette here - its the person that poses the question that gets to resolve the question, not the person answering